docs/UserGuide/API/Programming-JDBC.md

```diff
@@ -21,6 +21,10 @@
 
 ## JDBC
 
+*NOTICE: CURRENTLY, JDBC IS USED FOR CONNECTING SOME THIRD-PART TOOLS. 
+IT CAN NOT PROVIDE HIGH THROUGHPUT FOR WRITE OPERATIONS. 
+PLEASE USE [JAVA NATIVE API](Programming-Native-API.md) INSTEAD*
```

Comment:
Use the website url here?
https://iotdb.apache.org/UserGuide/Master/API/Programming-Java-Native-API.html